kubeflow/trainer/rhai/transformers.py

```
@@ -463,6 +471,195 @@ def relative_update(self, inc=1):
+        def _upload_worker_loop(self) -> None:
+            """Upload worker loop."""
+            while not self._shutdown_event.is_set():
+                try:
+                    # Use timeout to periodically check shutdown event
+                    task = self._upload_queue.get(timeout=1.0)
+                except Empty:
+                    continue
+
+                try:
+                    self._upload_checkpoint_to_cloud(task)
+                except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)